apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -0,0 +1,50 @@
+import { DateInput, PageLayout } from "~/libs/components/components.js";
+import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+const getDefaultDateRange = (): Date[] => {
+	const today = new Date();
+	const startOfMonth = new Date(today.getFullYear(), today.getMonth());
+
+	return [startOfMonth, today];
+};
```

Comment:
let's use last 30 days as default range, also we have a library for working with dates and we need to use it in case we need manipulations with the dates
```suggestion
```